16_security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,622 @@
+
+=== Unannounced Channels
+
+((("payment channel","unannounced channels")))((("security and privacy","unannounced channels")))((("unannounced channels")))If you intend to use the Lightning Network to send and receive funds between nodes and wallets you control, and have no interest in routing other users' payments, there is little need to announce your channels to the rest of the network.
+You could open a channel between, say, your desktop PC running a full node and your mobile phone running a Lightning wallet, and simply forgo the channel announcement discussed in <<ch03_How_Lightning_Works>>.
+These are sometimes called "private" channels; however, it is more correct to refer to them as "unannounced" channels because they are not strictly private.
+
+Unannounced channels will not be known to the rest of the network and won't normally be used to route other users' payments.
+They can still be used to route payments if other nodes are made aware of them; for example, an invoice could contain routing hints which suggests a path with an unannounced channel.
+However, assuming that you've only opened an unannounced channel with yourself, you do gain some measure of privacy.
+Since you are not exposing your channel to the network, you lower the risk of a denial-of-service attack on your node.
+You can also more easily manage the capacity of this channel, since it will only be used to receive or send directly to your node.
+
+There are also advantages to opening an unannounced channel with a known party that you transact with frequently.
+For example, if Alice and Bob frequently play poker for bitcoin, they could open a channel to send their winnings back and forth.
+Under normal conditions, this channel will not be used to route payments from other users or collect fees.
+And since the channel will not be known to the rest of the network, any payments between Alice and Bob cannot be inferred by tracking changes in the channel's routing capacity.
+This confers some privacy to Alice and Bob; however, if one of them decides to make other users aware of the channel, such as by including it in the routing hints of an invoice, then this privacy is lost.
+
+It should also be noted that to open an unannounced channel, a public transaction must be made on the Bitcoin blockchain.
+Hence it is possible to infer the existence and size of the channel if a malicious party is monitoring the blockchain for channel opening transactions and attempting to match them to channels on the network.
+Furthermore, when the channel is closed, the final balance of the channel will be made public once it's committed to the Bitcoin blockchain.
+However, since the opening and commitment transactions are pseudonymous, it will not be a simple matter to connect it back to Alice or Bob.
+In addition, the Taproot update of 2021 makes it difficult to distinguish between channel opening and closing transactions and other specific kinds of Bitcoin transactions.
```

Comment:
What specifically does Taproot add that makes it difficult to distinguish between channel opening and closing transactions and other types of transactions? While this might be a question better served by https://github.com/bitcoinbook/bitcoinbook, at the time of this comment it doesn't cover taproot whatsoever.

I'd suggest adding something like: 
```suggestion
In addition, the Bitcoin Taproot update of 2021 introduces a new SegWit output type that limits on-chain  transaction spendability information at creation or when spent. One of the ways it accomplishes this is by merging pay-to-pubkey and pay-to-scripthash policies, making all outputs spendable by either a key or a script and indistinguishable from each other. As long as the key-based spending path is used for spending, it is not revealed whether a script path was permitted as well, resulting in an increase in scripting privacy at spending time. Taproot also introduces  [Schnorr signatures](https://eprint.iacr.org/2018/068), which have a nice property of key aggregation: a single public key can be derived from the individual public keys of the signers. This makes M-of-N policies and signatures indistinguishable from single-party equivalents.
```